app/controllers/pieces_controller.rb

```diff
@@ -34,5 +44,15 @@ def set_game
     def piece_params
       params.require(:piece).permit(:id, :x_coordinate, :y_coordinate, :type)
     end
+    
+    def refresh_game
+      render json: {
+        redraw_game_url: game_path(@game)
+      }
+    end
+    
+    def set_game_winner
+      @game.update_attribute(:winning_player_id, current_player.id)
+    end
   
```

Comment:
I disagree. I know of developers who use this technique to make private methods easy to identify.